source/reference/json/search-nodes-config-file.txt

```diff
@@ -0,0 +1,92 @@
+.. _atlas-cli-search-nodes-config-file:
+
+===============================
+Search Nodes Configuration File
+===============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can use a configuration file to specify the required 
+settings for creating search nodes on a {+cluster+} 
+by using the {+atlas-cli+}. The {+atlas-cli+} accepts 
+``.json`` configuration files. 
```

Comment:
Suggestion:

To create or update search nodes in a {+cluster+} you can
create a ``.json`` configuration file for search nodes and pass this file to a command with {+atlas-cli+}.